ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -264,6 +275,34 @@ private void validateConfig(final JsonRpcConfiguration config) {
     return resultFuture;
   }
 
+  private Handler<HttpConnection> connectionHandler() {
+
+    return connection -> {
+      if (activeConnectionsCount.get() >= maxActiveConnections) {
+        LOG.warn(
+            "Max Active Connections limit {} reached. Rejecting all new connections.",
+            maxActiveConnections);
+        // disallow new connections to prevent DoS
+        LOG.warn(
+            "Rejecting new client connection: remoteAddress: {} connection count: {}",
+            connection.remoteAddress(),
+            activeConnectionsCount.incrementAndGet());
```

Comment:
We should probably log only one of these msgs. I believe we don't the two.

I think we can keep the msg when we reject the connection and change the msg slightly:
```
Rejected new HTTP connection from 1921.68.0.1. Max 80 active HTTP connections limit reached.
```
(or something similar)